### PR TITLE
✨Add Krantz-CDL-FY21-RNA constants to kflib

### PR DESCRIPTION
**Pull request by r3m0chop**
New ingest constants as for the Krantz-CDL-FY21-RNA clinical data, thank you very much!
cc: M1 team: @znatty22 , @chris-s-friedman , @youngnm , Meen, etc
cc: KF-DRC: @Christina-J-Diaz , @HigginsD3b, @aw3334

### CODE REVIEW

**Review by chris-s-friedman — DISMISSED** [review]
Looks good!

File: kf_lib_data_ingest/common/constants.py [hunk 1]
```diff
@@ -364,6 +365,10 @@ class CONSENT_TYPE:
     # General Research Use (NPU) (GRU-NPU) - Use of this data is
     # limited to general research use. Use of the data is limited to
     # not-for-profit organizations.
+    HMB_GSO = "HMB-GSO”
```
Comment (chris-s-friedman) [inline kf_lib_data_ingest/common/constants.py:368]: Looks like the closing quote is breaking CI + linting. I think this should fix it
```suggestion
    HMB_GSO = "HMB-GSO"
```

**Review by chris-s-friedman — CHANGES_REQUESTED** [review]
Fix that closing `"` and once this passes tests, I think you'll be good to go.

File: kf_lib_data_ingest/common/constants.py (hunk 1 above)
Comment (r3m0chop) [reply kf_lib_data_ingest/common/constants.py:368]: funky "smart quotes" might have gotten in there.  I thought that I had resolved it in the opening quote for the tests to pass, but seems that closing quote was still trying to be smart.  Thanks for the good eye, Chris!

Comment (r3m0chop) [reply kf_lib_data_ingest/common/constants.py]: unsmarted the quote in https://github.com/kids-first/kf-lib-data-ingest/pull/658/commits/028e6836d6cddaa73bb8e8698ee0d0bda23bf538 ;-)

**Review by Christina-J-Diaz — DISMISSED** [review]
Looks good :)

**Comment by r3m0chop** [comment]
commits squashed :-)

**Review by Christina-J-Diaz — APPROVED** [review]

**Review by chris-s-friedman — APPROVED** [review]
looks great!

**Pull request merged**